src/index.ts

```diff
@@ -62,6 +62,6 @@ if (configJSON) {
   initReporters(config);
 }
 
-const commandOptions = process.argv.slice(2).filter((option) => option.indexOf(configPath as string) >= 0);
+const commandOptions = process.argv.slice(2);
```

Comment:
Hi @roman-wedemeier . Thanks for the PR, but as you see from the CI result `jasmine` has strict parameter validation. You have to filter the `commandOptions` that contain only allowed parameters.

Please don't forget write test that validate positive and negative cases too.

If you think I will implement it tomorrow.